pages/common/nslookup.md

```diff
@@ -0,0 +1,27 @@
+# nslookup
+
+> Query name server(s) for various domain records.
+
+- Query your system default name server for an IP address (A record) of the domain:
+
+`nslookup  {{example.com}}`
+
+- Query non default name server (here Google DNS 8.8.8.8) for a NS record of the domain:
+
+`nslookup type=NS {{example.com}} 8.8.8.8`
+
+- Query default name server for an IP address (A record) of the domain using TCP protocol instead of the default UDP
+
+`nslookup -vc -type=A {{example.com}}`
```

Comment:
What does v and c stand for? And why does -type=A need to be specified here but not in the first example?